### PR TITLE
Add swap space in EC2 instances

### PR DESCRIPTION
**Pull request by henriqueribeiro**
*Issue #, if available:*
https://github.com/aws-samples/aws-genomics-workflows/issues/124

*Description of changes:*
- Add 4GB of swap space in EC2 instances

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/templates/gwfcore/gwfcore-launch-template.template.yaml [hunk 1]
```diff
@@ -103,6 +103,14 @@ Resources:
                 - unzip
 
                 runcmd:
+                # add 4GB of swap space
```
Comment (wleepang) [inline src/templates/gwfcore/gwfcore-launch-template.template.yaml:106]: Prefer to move these lines to `ecs-additions-common.sh` and keep the LaunchTemplate UserData as minimal as possible. When the GWFCore stack is launched the `ecs-additions-*` scripts are "installed" in a CodeCommit repo. Updates to the repo trigger redeployment to an artifacts location that referenced by the LaunchTemplate. This enables users to customize provisioning steps without having to teardown and relaunch the GWFCore stack.

**Review by wleepang — CHANGES_REQUESTED** [review]
Suggest moving the swap provisioning to `ecs-additions-common.sh`

File: src/templates/gwfcore/gwfcore-launch-template.template.yaml (hunk 1 above)
Comment (henriqueribeiro) [reply src/templates/gwfcore/gwfcore-launch-template.template.yaml:106]: Makes total sense. I will update it

**Review by wleepang — APPROVED** [review]
Looks good!

**Pull request merged**